### PR TITLE
Added a option to quit with q or esc

### PR DESCRIPTION
**Pull request by toniz4**
I never worked with gtk, and probably that is not the right implementation, but it works.

### CODE REVIEW

File: dragon.c [hunk 1]
```diff
@@ -401,17 +401,20 @@ int main (int argc, char **argv) {
...
 
     window = gtk_window_new(GTK_WINDOW_TOPLEVEL);
 
-    closure = g_cclosure_new(G_CALLBACK(do_quit), NULL, NULL);
+    closure1 = g_cclosure_new(G_CALLBACK(do_quit), NULL, NULL);
+    closure2 = g_cclosure_new(G_CALLBACK(do_quit), NULL, NULL);
     accelgroup = gtk_accel_group_new();
-    gtk_accel_group_connect(accelgroup, GDK_KEY_Escape, 0, 0, closure);
+    gtk_accel_group_connect(accelgroup, GDK_KEY_Escape, 0, 0, closure1); 
+	gtk_accel_group_connect(accelgroup, GDK_KEY_q, 0, 0, closure2);
```
Comment (mwh) [inline dragon.c:417]: The indentation seems off here.

```suggestion
    gtk_accel_group_connect(accelgroup, GDK_KEY_q, 0, 0, closure2);
```

File: dragon.c [hunk 2]
```diff
@@ -401,17 +401,20 @@ int main (int argc, char **argv) {
...
 
     gtk_init(&argc, &argv);
 
     icon_theme = gtk_icon_theme_get_default();
 
     window = gtk_window_new(GTK_WINDOW_TOPLEVEL);
 
-    closure = g_cclosure_new(G_CALLBACK(do_quit), NULL, NULL);
+    closure1 = g_cclosure_new(G_CALLBACK(do_quit), NULL, NULL);
+    closure2 = g_cclosure_new(G_CALLBACK(do_quit), NULL, NULL);
```
Comment (mwh) [inline dragon.c:414]: Is there a reason we can't just use the closure more than once? I wouldn't have thought so, but perhaps there is.

Comment (toniz4) [reply dragon.c:414]: when using the same closure i get this error
``(dragon:15147): Gtk-CRITICAL **: 13:14:54.912: gtk_accel_group_connect: assertion 'gtk_accel_group_from_accel_closure (closure) == NULL' failed
```
and i can't exit with q

Comment (toniz4) [reply dragon.c:414]: redeclaring closure before the second acel group seems to work too, don't know if is elegant tho

**Comment by cljoly** [comment]
This is a cool feature, thanks @toniz4!

**Comment by toniz4** [comment]
unfortunately i think that this repo is abandoned

**Comment by mwh** [comment]
For reasons I don't understand GitHub won't let me merge this, due to "conflicts" it won't let me fix, even though it's a two-line difference. I have manually squashed it all into a single commit and pushed that.

**Review by mwh — APPROVED** [review]

**Comment by mwh** [comment]
Well, I've done what I can - the final code looks like your code, and I took over your first commit, but it's strictly unrelated so it doesn't get detected as resolving this PR. It is accepted & merged. Thanks!

(oddly, now it *will* let me resolve a merge conflict, but only a new whitespace one I introduced in the squash, not whatever the underlying issue is)

**Pull request closed without merging**